examples/210-embedding_models.ipynb

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
<p>"this" -&gt; "these"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>